### PR TITLE
List Posts functionality implemented

### PR DESCRIPTION
**Pull request by hcrudge**
Updated to add a .gitignore, create post list functionality.
Created PostManager.cs.
Modified MainMenuManager.cs to link to the PostManager.cs
Modified PostRepository.cs to create GetAll() method.

### CODE REVIEW

File: TabloidCLI/Repositories/PostRepository.cs [hunk 1]
```diff
@@ -2,16 +2,51 @@
 using System.Collections.Generic;
 using Microsoft.Data.SqlClient;
 using TabloidCLI.Models;
+using TabloidCLI.Repositories;
 
-namespace TabloidCLI.Repositories
+
+namespace TabloidCLI
```
Comment (joshdbarton) [inline TabloidCLI/Repositories/PostRepository.cs]: Curious why the change on the namespace here?

Comment (hcrudge) [reply TabloidCLI/Repositories/PostRepository.cs]: I noticed this was done in the author repository, so I did it in the post repository for consistency. But it could be updated back to what it initially was, and I can refactor.

**Comment by sxiong0519** [comment]
💯

**Comment by mtrevin93** [comment]
👍

File: TabloidCLI/Repositories/PostRepository.cs (hunk 1 above)
Comment (joshdbarton) [reply TabloidCLI/Repositories/PostRepository.cs]: Seems like an oversight... if it's a repository, it makes sense that all of the repositories would be in the Repositories namespace.

Comment (hcrudge) [reply TabloidCLI/Repositories/PostRepository.cs]: Just reviewed, and it looks like the pre-populated TagsRepository has the same format  as the prepopulated AuthorsRepository. I'll update the Posts to be within the Repositories namespace.

Comment (joshdbarton) [reply TabloidCLI/Repositories/PostRepository.cs]: yeah, there are a number of the repos that seem to be like that.

**Comment by pracinejr** [comment]
💯
